### ISSUE TITLE
Can't run "rvm-mini -i" due outdated rubygems and SSL error

### ISSUE THREAD

**Issue by TomFreudenberg** [4commerce-technologies-AG/rvm-mini#3]
When using e.g. rvm-mini and latest (current 2.2.5) rvm binaries for MAC OSX, you will get an error when `gem` tries to install `bundler` like:

```
bin/rvm-mini --ruby-install

Start ruby installation ...

Get ruby package for os/machine type osx/x86_64 distribution release 10.11.
load ruby binaries from: https://rvm.io/binaries/osx/10.11/x86_64/ruby-2.2.5.tar.bz2

Check bundler ... bundle seems to be not installed!
now installing ...

ERROR:  Could not find a valid gem 'bundler' (>= 0), here is why:
          Unable to download data from https://rubygems.org/ - SSL_connect returned=1 errno=0 state=error: certificate verify failed (https://api.rubygems.org/specs.4.8.gz)

I am sorry, there was an error during bundler installation. Please retry. Abort!
```

This happens while the packed rvm gem version is outdated due openssl libraries. For that you need to update rubygems.

Instead of doing a complicated manual or insecure rubygems update, we add the feature option `--rubygems-update [2.6.7]` to the installer.

**Comment by TomFreudenberg** [comment]
Checkout

https://rubygems.org/pages/download?locale=en

for latest rubygems releases and information how to install manually.

**Comment by TomFreudenberg** [comment]
Done and published as release/1.1
